registry.k8s.io/images/k8s-staging-kops/images.yaml

```diff
@@ -1632,3 +1632,18 @@
   dmap:
     "sha256:250d47eb7eed28636dedb2ae7d1d4217db4657e16dbfe5a9e91a137ae386f2b8": ["1.32.0-beta.1"]
 
+
+# 1.32.0
```

Comment:
Looks like I'm going to have to rebase at least one of these image PRs :-(

Reply:
exactly...

Reply:
@justinsb ready for rebase

Reply:
Thanks - rebased!

Reply:
/lgtm